packages/start-slicemachine/src/lib/createSliceMachineServer.ts

```diff
@@ -79,6 +85,22 @@ export const createSliceMachineServer = async (
 
 	router.add("/api/t", eventHandler(sentryHandler));
 
+	router.add(
+		"/api/crash",
+		eventHandler(() => {
+			throw new Error("Uncaught Error");
+		}),
+	);
```

Comment:
What is this endpoint used for? If we are catching errors and re-throwing this custom "Uncaught Error", are we losing metadata from the originally thrown error somewhere in the process?

Reply:
I assume we can delete it now? @mathieud